docs/guides/setup/contributing-with-github-desktop.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to Carbon using Github Desktop
+
+## Initial Setup
+
+1. Download and install [Github Desktop](https://desktop.github.com/)
+2. Run the Github Desktop application
+3. Click the button labeled 'Sign in to Github.com'. This will open your browser and take you to an authorization page. 
+4. Click the 'Authorize desktop' button, which may prompt you for your Github.com password
+5. You will be asked to configure your Git installation. Set your username and email then click the button labeled 'Continue'
+6. Agree or disagree to periodic usage stats
+7. Navigate to the URL of the repository you'd like to contribute to. For example [Carbon Design System](https://github.com/carbon-design-system/carbon)
+8. In the top right of the page click the button labeled 'Fork'. This will create a copy of the repository and add it to your account. This is where you'll make changes and later request the repository you just forked pull those changes from (more on that later).
+9. Back in the Github Desktop application click the button labeled 'Clone a repository from the internet...'
+10. In the modal that pops up you can search your account for the fork you just made. Once you've selected the right repository click the blue button labeled 'Clone...' in the bottom right corner of the modal.
+11. The _next_ modal is a confirmation screen where you can double check the URL of your fork and select where on your computer you'd like to clone the repository. 
+12. Once the Github Application is done cloning your repository select that you intend 'To contribute to the parent project' and then click the button labeled 'Continue'. You are now ready to work.
+
+## Basic workflow
+
+1. First things first we need to create a "working" branch. At the top of the Github Desktop application click the drop down labeled 'Current branch'
+2. Click the button labeled 'New branch' and then name your branch. Typically this will be a general description of the work youre doing. Maybe 'updating-accessibility-docs' or 'adding-security-pictograms'. Then click 'Create branch' to continue
```

Comment:
`you’re` Is just missing the apostrophe.

Reply:
Honestly pleasantly surprised that was the only obvious grammatical error.